API_Auto/src/test/java/postAllProduct/AllProduct.java

```diff
@@ -26,22 +26,18 @@ public AllProduct() {
 
     @Test(priority = 1)
     public void checkStatusCode() {
-        Response response = RestAssured.post(url)
-                .then().extract().response();
+        Response response = RestAssured.post(url).then().extract().response();
         Assert.assertEquals(response.getStatusCode(), 200);
-        System.out.println("The response code is : " + response.getStatusCode());
         log.info("The response code is : " + response.getStatusCode());
         
     }
 
     @Test(priority = 2)
     public void checkResponse() {
-        Response response = RestAssured.post(url)
-                .then().extract().response();
+        Response response = RestAssured.post(url).then().extract().response();
```

Comment:
Reuse the existing code that has been implemented in API Assignment 1

Reply:
done